### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by VicPeralta**
# Math magicians: components
In this milestone:
- Made sure there are no linter errors
- Made sure the correct GitFlow was used
- Repo documented in a professional way
- Delete all boilerplate created from CRA
- Create a directory `components`
- Inside `components` a new file name `calculator.js` was created
- In `calculator.js` a React component was created that matches the design provided.
- Developed just the UI, no logic was added
- Tested it in the Browser and it is rendered without a problem

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const Calculator = () => (
+  <div className="calculator">
+    <ResultWindow />
+    <KeyRow keys={['AC', '+/-', '%', '÷']} />
+    <KeyRow keys={['7', '8', '9', 'X']} />
+    <KeyRow keys={['4', '5', '6', '-']} />
+    <KeyRow keys={['1', '2', '3', '+']} />
+    <KeyRow keys={['0', '.', '=']} />
+  </div>
+);
```
Comment (FrederickMih) [inline src/components/calculator.js]: - [x] What a fantastic component! :100: . I can see that you have a good mastering of React functional-based components. Please let's just make it a class-based component for the moment as the requirements of this project stimulate.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
```
Comment (FrederickMih) [inline src/App.js]: - [x] Hi @VicPeralta, you are doing great on this project :+1: . Please let's transform this into a class-based component as the requirements of this project outline.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @VicPeralta , 

You are doing so great on this project so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


# To Highlights

- Your repo looks professional :heavy_check_mark: 


## Required Changes ♻️

- [x] Your README file is well structured and descriptive. Would you include a link to your LinkedIn and Twitter account?. This can enable recruiters to know more about you. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @VicPeralta, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**